client-sdk/mft-python-sdk/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
You don't need additional license files as it's already in the root directory

Reply:
Since these Client SDKs going to be individual components should we not include LICENSE? We followed the Python SDK format that was written for Custos. We had those files there. Seeking your confirmation again to remove the LICENSE file.

Reply:
@gkiran292 Yes you have a point. Let's keep the license file as it is.